pandaserver/brokerage/broker.py

```diff
@@ -243,30 +243,30 @@ def sendMsgToLoggerHTTP(msgList, job):
 
 
 # get T2 candidates when files are missing at T2
-def getT2CandList(tmpJob, siteMapper, t2FilesMap):
```

Comment:
I think we concluded in the discussion that T1s-> nuclei and T2s->satellites? With your changes we are entering a mixed nomenclature mode using both together.

Reply:
Will replace it, it's just that it was on broker.py and not on the dataservice folder. I guess you prefer to have it corrected also there, I will update my PR